pkgs/development/interpreters/python/default.nix

```diff
@@ -119,15 +119,6 @@ with pkgs;
   };
 
   sources = {
-    python38 = {
-      sourceVersion = {
-        major = "3";
-        minor = "8";
-        patch = "12";
-        suffix = "";
-      };
-      sha256 = "1si8hw2xpagh4iji89zdx69p3dv5mjqwwbx2x2sl6lrp41jaglxi";
-    };
```

Comment:
Ah wait. When do we remove versions from the sources attrset again? Isn't this where we track the two currently supported ones? I don't know.

Reply:
They are referenced in the same file. The reason the source details were split into separate sets is for the minimal build for bootstrapping.